ppxlib.opam

```
@@ -14,7 +14,13 @@ run-test: [
   ["dune" "runtest" "-p" name "-j" jobs] { ocaml:version >= "4.07" }
 ]
 depends: [
-  "ocaml"                   {>= "4.04.1" & < "4.13"}
+  (
+    ("ocaml" {>= "4.04.1" & < "4.13"}) |
```

I think it might need to be:
```suggestion
    ("ocaml" {>= "4.04.1" & < "4.07"}) |
```
or the with-test can choose to ignore the cinaps

---

ouh yeah my bad!